packages/serve/src/index.ts

```diff
@@ -5,6 +5,25 @@ class ServeCommand {
     async apply(cli: any): Promise<void> {
         const { logger } = cli;
 
+        const loadDevServerOptions = () => {
+            // eslint-disable-next-line @typescript-eslint/no-var-requires, node/no-extraneous-require
+            const options = require('webpack-dev-server/bin/cli-flags');
+
+            // Old options format
+            // { devServer: [{...}, {}...] }
+            if (options.devServer) {
+                return options.devServer;
+            }
+
+            // New options format
+            // { flag1: {}, flag2: {} }
+            return Object.keys(options).map((key) => {
```

Comment:
We will avoid it after switch on object options (part of perf)

Reply:
yep thanks